src/pages/Home/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+import MovieDetail from '../../components/MovieDetail';
+import MovieList from '../../components/CardList';
+import { getSelectedMovie, getGenresList } from '../../services';
+
+function Home() {
+  const [movieInfo, setMovieInfo] = useState(null);
+  const genresLabelList = getGenresList();
+
+  useEffect(() => {
+    setMovieInfo(getSelectedMovie());
+  }, []);
+
+  let configParams = {
+    page: '1',
+    isPoster: true,
+  };
+
+  return (
+    <div>
+      { (movieInfo !== null)
```

Comment:
we can set an if before the return of the component with to check if there is no moveInfo and in that case return null